### ISSUE TITLE
E_WARNING in libraries/joomla/session/handler/joomla.php - J3.6.2

### ISSUE THREAD

**Issue by cavo789** [joomla/joomla-cms#12153]
Hello guys

In my PHP application, I'm interfacing the Joomla!® framework for retrieving the list of versions and being able to suggest to my users that their version is an old one.  Quite easy.  

By loading the framework; I receive this warning : 
`[E_WARNING] [2016-09-23 13:32:27 (CEST)] [File xxxx/libraries/joomla/session/handler/joomla.php] [Line 45] ==> ini_set(): A session is active. You cannot change the session module's ini settings at this time`

This happens because, in my own code, I've already started a session.

Could be easily solved by checking session_id().
### Steps to reproduce the issue

The current code in /libraries/joomla/session/handler/joomla.php is :

``` php
public function __construct($options = array())
{
   // Disable transparent sid support
   ini_set('session.use_trans_sid', '0');

   // Only allow the session ID to come from cookies and nothing else.
   ini_set('session.use_only_cookies', '1');

   // Set options
   $this->setOptions($options);
   $this->setCookieParams();
}
```
### Expected result

No warning
### Actual result

A warning is displayed : 
`[E_WARNING] [2016-09-23 13:32:27 (CEST)] [File xxxx/libraries/joomla/session/handler/joomla.php] [Line 45] ==> ini_set(): A session is active. You cannot change the session module's ini settings at this time`
### Additional comments

By changing the code like below, the warning dissapear.  Can this change be done in Joomla ?  Don't know if there will be side effect.

``` php
public function __construct($options = array())
{
   if (session_id()==0)
   {
      // Disable transparent sid support
      ini_set('session.use_trans_sid', '0');

      // Only allow the session ID to come from cookies and nothing else.
      ini_set('session.use_only_cookies', '1');
   }

   // Set options
   $this->setOptions($options);
   $this->setCookieParams();
}
```

Thanks a lot.

**Comment by wdj-ac** [comment]
Get the same errors in my logs. 
Joomla 3.6.5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12153">issues.joomla.org/tracker/joomla-cms/12153</a>.</sub>

**Comment by PhilETaylor** [comment]
Please test https://github.com/joomla/joomla-cms/pull/15742

**Comment by franz-wohlkoenig** [comment]
closed as having PR #12153<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12153">issues.joomla.org/tracker/joomla-cms/12153</a>.</sub>
